src/sadie/anarci/aa/_anarci.py

```diff
@@ -66,7 +66,7 @@
 import warnings
 from Bio.SearchIO.HmmerIO import Hmmer3TextParser as HMMERParser
 
-from sadie.anarci.hmmer import HMMER
+# from sadie.anarci.hmmer import HMMER
```

Comment:
run_hmmer and check_for_j and anything downstream was deprecated. Will be removing it in a doc/clean PR